packages/backend-common/src/cache/types.ts

```diff
@@ -25,6 +26,21 @@ type ClientOptions = {
   defaultTtl?: number;
 };
 
+export type OptionalOnError = ((err: Error) => void) | undefined;
+
+export type CacheManagerOptions = {
```

Comment:
just noting that this isn't exported in the index file, so it isn't visible publicly (see the api-report, where it is in use but not declared)